raiden/ui/cli.py

```diff
@@ -942,27 +943,37 @@ def _run_app():
     # TODO:
     # - Ask for confirmation to quit if there are any locked transfers that did
     # not timeout.
-    if kwargs['transport'] == 'udp':
-        (listen_host, listen_port) = split_endpoint(kwargs['listen_address'])
-        try:
-            with SocketFactory(listen_host, listen_port, strategy=kwargs['nat']) as mapped_socket:
-                kwargs['mapped_socket'] = mapped_socket
-                app_ = _run_app()
+    try:
+        if kwargs['transport'] == 'udp':
+            (listen_host, listen_port) = split_endpoint(kwargs['listen_address'])
+            try:
+                with SocketFactory(
+                        listen_host, listen_port, strategy=kwargs['nat'],
+                ) as mapped_socket:
+                    kwargs['mapped_socket'] = mapped_socket
+                    app_ = _run_app()
 
-        except RaidenServicePortInUseError:
-            print(
-                'ERROR: Address %s:%s is in use. '
-                'Use --listen-address <host:port> to specify port to listen on.' %
-                (listen_host, listen_port),
-            )
-            sys.exit(1)
-    elif kwargs['transport'] == 'matrix':
-        kwargs['mapped_socket'] = None
-        app_ = _run_app()
-    else:
-        # Shouldn't happen
-        raise RuntimeError(f"Invalid transport type '{kwargs['transport']}'")
-    app_.stop(leave_channels=False)
+            except RaidenServicePortInUseError:
+                print(
+                    'ERROR: Address %s:%s is in use. '
+                    'Use --listen-address <host:port> to specify port to listen on.' %
+                    (listen_host, listen_port),
+                )
+                sys.exit(1)
+        elif kwargs['transport'] == 'matrix':
+            kwargs['mapped_socket'] = None
+            app_ = _run_app()
+        else:
+            # Shouldn't happen
+            raise RuntimeError(f"Invalid transport type '{kwargs['transport']}'")
+        app_.stop(leave_channels=False)
+    except ReplacementTransactionUnderpriced:
+        print(
+            'It looks like the same account is being used by different '
```

Comment:
Does that only happen when the account is used independently of raiden? It could also happen that the tx wasn't mined, but then `eth.gasPrice` gets lower and another tx is sent!?

Reply:
Not necessarily. I faced this problem when my node crashed and i restarted the node causing this issue to appear. For some reason, a replacement transaction was being sent while the previous one wasn't yet mined while the replacement transaction had <= the gas amount the previous transaction had.

Reply:
In general, we currently assume that if this happens again, then it would most probably be an issue caused by using the same account somewhere else other than the raiden node.